int/api/cmd/deploySC.go

```diff
@@ -13,6 +15,9 @@ import (
 	"github.com/massalabs/station/pkg/onchain"
 )
 
+//go:embed sc
+var content embed.FS
```

Comment:
You can directly embed the SC like: 
```suggestion
//go:embed sc/deployer.wasm
var deployerSC []byte
```

Like that, you don't need to do the `content.ReadFile` line 31